zerver/lib/upload.py

```diff
@@ -74,7 +74,7 @@ class RealmUploadQuotaError(JsonableError):
     code = ErrorCode.REALM_UPLOAD_QUOTA
 
 def attachment_url_to_path_id(attachment_url: str) -> str:
```

Comment:
This might want a new name, since the thing it is getting is a path, maybe `attachment_path_to_path_id`?